PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -269,6 +269,10 @@ class PsfPowershellScriptRunner
 					SSWrapper = file.path();
 				}
 			}
+			if (!std::filesystem::exists(SSWrapper))
+			{
+				MessageBoxEx(NULL, L"'StartingScriptWrapper.ps1' was not found. This is required to run the PowerShell scripts through PSF. We recommend you to add it under the package root folder and re-install the application.", L"MSIX Packet Support Framework", MB_OK | MB_ICONWARNING, 0);

```

Comment:
MessageBoxEx does not stop execution.  PSF will continue to execute even if starting script wrapper can't be found.

What is the expected behavior if the script can't be found?  Stop PSF, issue a warning but keep going?

I recommend to stop execution.  Apps need this script to run before start up and apps expect the script to be ran.  PSF should halt if it can't run the powershell script.

Reply:
In many case, even through the PowerShell script fails, it is observed that we can go and start the actual application. Hence we are not making it mandatory to halt PSF but continuing to execute the actual application by giving a warning.

Reply:
Correct.  I'm worried about state not being setup and the app either behaving differently.

Can you please add something like "The app will continue to run but the app might not behave as expected."?  To let the user know what might happen?